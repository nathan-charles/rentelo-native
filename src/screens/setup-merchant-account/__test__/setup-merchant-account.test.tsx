import React from 'react';
import renderer from 'react-test-renderer';
import { MockedProvider } from '@apollo/react-testing';

import SetupMerchantAccount from '../';

test('renders correctly', () => {
  const navigation = { navigate: jest.fn() };
  const tree = renderer
    .create(
      <MockedProvider mocks={[]} addTypename={false}>
        <SetupMerchantAccount navigation={navigation} />
      </MockedProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
