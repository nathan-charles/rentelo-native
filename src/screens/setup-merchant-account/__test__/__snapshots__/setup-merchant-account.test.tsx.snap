// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <View
        accessible={true}
        focusable={false}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
      >
        <View
          itemDivider={true}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#f4f4f4",
              "borderBottomWidth": null,
              "borderColor": "#c9c9c9",
              "flexDirection": "row",
              "marginLeft": null,
              "paddingBottom": 10,
              "paddingLeft": 15,
              "paddingRight": 10,
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "alignSelf": "center",
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              }
            }
            uppercase={false}
          >
            PERSONAL INFORMATION
          </Text>
        </View>
      </View>
      <View
        padder={true}
        style={
          Object {
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              FIRST NAME
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="words"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholder="John"
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="Nathan"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              LAST NAME
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="words"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholder="Doe"
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="Charles"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              DATE OF BIRTH
            </Text>
            <TextInput
              allowFontScaling={true}
              customTextInput={[Function]}
              editable={true}
              keyboardType="number-pad"
              onBlur={[Function]}
              onChangeText={[Function]}
              options={
                Object {
                  "format": "MM/DD/YYYY",
                }
              }
              placeholder="MM/DD/YYYY"
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              type="datetime"
              underlineColorAndroid="rgba(0,0,0,0)"
              value="12/22/1984"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              SSN
            </Text>
            <TextInput
              allowFontScaling={true}
              customTextInput={[Function]}
              editable={true}
              keyboardType="number-pad"
              onBlur={[Function]}
              onChangeText={[Function]}
              options={
                Object {
                  "getRawValue": [Function],
                  "mask": "9999",
                }
              }
              placeholder="000-00-0000"
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              type="custom"
              underlineColorAndroid="rgba(0,0,0,0)"
              value="7631"
            />
          </View>
          <Text
            note={true}
            style={
              Array [
                Object {
                  "color": "#a7a7a7",
                  "fontFamily": "System",
                  "fontSize": 14,
                },
                Object {
                  "marginVertical": 4,
                },
              ]
            }
            uppercase={false}
          >
            We need your SSN to verify your identity, we do not store this it's used by our payment processor.
          </Text>
        </View>
      </View>
      <View
        accessible={true}
        focusable={false}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
      >
        <View
          itemDivider={true}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#f4f4f4",
              "borderBottomWidth": null,
              "borderColor": "#c9c9c9",
              "flexDirection": "row",
              "marginLeft": null,
              "paddingBottom": 10,
              "paddingLeft": 15,
              "paddingRight": 10,
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "alignSelf": "center",
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              }
            }
            uppercase={false}
          >
            CONTACT
          </Text>
        </View>
      </View>
      <View
        padder={true}
        style={
          Object {
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              EMAIL
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              editable={true}
              keyboardType="email-address"
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholder="john.doe@me.com"
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="nathan.charles@me.com"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              PHONE
            </Text>
            <TextInput
              allowFontScaling={true}
              customTextInput={[Function]}
              editable={true}
              keyboardType="number-pad"
              onBlur={[Function]}
              onChangeText={[Function]}
              options={
                Object {
                  "getRawValue": [Function],
                  "mask": "(999) 999-9999",
                }
              }
              placeholder="(000) 000-0000"
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              type="custom"
              underlineColorAndroid="rgba(0,0,0,0)"
              value="(214) 986-5391"
            />
          </View>
        </View>
      </View>
      <View
        accessible={true}
        focusable={false}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
      >
        <View
          itemDivider={true}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#f4f4f4",
              "borderBottomWidth": null,
              "borderColor": "#c9c9c9",
              "flexDirection": "row",
              "marginLeft": null,
              "paddingBottom": 10,
              "paddingLeft": 15,
              "paddingRight": 10,
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "alignSelf": "center",
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              }
            }
            uppercase={false}
          >
            ADDRESS
          </Text>
        </View>
      </View>
      <View
        padder={true}
        style={
          Object {
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              ADDRESS 1
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="words"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="1317 Mary Horn Ln"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              ADDRESS 2
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="words"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value=""
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              CITY
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="words"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="Savannah"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              STATE
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="words"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="TX"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              ZIP
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="76227"
            />
          </View>
        </View>
      </View>
      <View
        accessible={true}
        focusable={false}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
      >
        <View
          itemDivider={true}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#f4f4f4",
              "borderBottomWidth": null,
              "borderColor": "#c9c9c9",
              "flexDirection": "row",
              "marginLeft": null,
              "paddingBottom": 10,
              "paddingLeft": 15,
              "paddingRight": 10,
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Object {
                "alignSelf": "center",
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              }
            }
            uppercase={false}
          >
            BANKING
          </Text>
        </View>
      </View>
      <View
        padder={true}
        style={
          Object {
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              ROUTING NUMBER
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              editable={true}
              keyboardType="email-address"
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="110000000"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              ACCOUNT NUMBER
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="000123456789"
            />
          </View>
        </View>
        <View
          style={
            Array [
              Object {},
              Object {
                "marginVertical": 4,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "backgroundColor": "#FFF",
                  "borderColor": "#DDD",
                  "borderRadius": 5,
                  "borderWidth": 1,
                  "paddingHorizontal": 10,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 17,
                  },
                  Object {
                    "fontSize": 10,
                    "fontWeight": "500",
                    "paddingTop": 6,
                  },
                ]
              }
            >
              VERIFY ACCOUNT NUMBER
            </Text>
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              autoCorrect={false}
              editable={true}
              onBlur={[Function]}
              onChangeText={[Function]}
              placeholderTextColor="#DCDCDC"
              rejectResponderTermination={true}
              selectionColor="#F63440"
              style={
                Array [
                  Object {
                    "color": "#000",
                    "flex": 1,
                    "fontSize": 17,
                    "height": 50,
                    "paddingLeft": 5,
                    "paddingRight": 5,
                  },
                  Object {
                    "height": 40,
                    "paddingLeft": 0,
                    "paddingRight": 0,
                  },
                ]
              }
              underlineColorAndroid="rgba(0,0,0,0)"
              value="000123456789"
            />
          </View>
        </View>
      </View>
      <View
        padder={true}
        style={
          Object {
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "stretch",
              "backgroundColor": "#007aff",
              "borderBottomWidth": null,
              "borderColor": "#007aff",
              "borderLeftWidth": null,
              "borderRadius": 5,
              "borderRightWidth": null,
              "borderTopWidth": null,
              "borderWidth": undefined,
              "elevation": 2,
              "flexDirection": "row",
              "height": 45,
              "justifyContent": "center",
              "opacity": 1,
              "paddingBottom": 6,
              "paddingTop": 6,
              "shadowColor": undefined,
              "shadowOffset": undefined,
              "shadowOpacity": undefined,
              "shadowRadius": undefined,
            }
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#fff",
                "fontFamily": "System",
                "fontSize": 16.5,
                "marginLeft": 0,
                "marginRight": 0,
                "paddingLeft": 16,
                "paddingRight": 16,
              }
            }
            uppercase={false}
          >
            SUBMIT
          </Text>
        </View>
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "color": "red",
              },
            ]
          }
          uppercase={false}
        />
      </View>
    </View>
  </RCTScrollView>
</View>
`;
